boards/imix/src/lowpan_frag_dummy.rs

```diff
@@ -0,0 +1,507 @@
+//! A dummy sixlowpan/IP sender
```

Comment:
They are fairly similar, but `sixlowpan_dummy.rs` only tests compression/decompression, whereas `lowpan_frag_dummy.rs` tests fragmentation and reassembly, while the lower layer (the implemented fragmentation layer) does the compression invisibly to the test suite. I think it is useful to have both types of tests as it would be easier to pinpoint whether a bug is in the compression/decompression routines versus the fragmentation layer, but they could probably be combined into a single file.